kafka/producer/src/main/java/Main.java

```diff
@@ -80,7 +71,7 @@ public static void main(String[] args) throws InterruptedException {
                 // Increment number of sent messages for non blocking producer
                 numSent.incrementAndGet();
             }
-            if (transactionalProducer && ((i + 1) % msgPerTx == 0)) {
+            if (transactionalProducer && (i + 1) % msgPerTx == 0) {
```

Comment:
Same as above, with previous code this was more readable and clear

Reply:
Checkstyle plugin ...